app/backend/src/couchers/urls.py

```diff
@@ -52,3 +52,11 @@ def media_upload_url(path):
 
 def change_email_link(confirmation_token):
     return f"{config['BASE_URL']}/confirm-email/{confirmation_token}"
+
+
+def donation_cancelled_url():
+    return f"{config['BASE_URL']}/donate?cancelled=true"
+
+
+def donation_success_url():
+    return f"{config['BASE_URL']}/donate?success=true&session={{CHECKOUT_SESSION_ID}}"
```

Comment:
What is the f"{{foo}}" supposed to mean?

Reply:
I want the final string to be something like `https://couchers.org/donate?success=true&session={CHECKOUT_SESSION_ID}`: then stripe will fill in the checkout session id

Reply:
Where is it documented? (Couldn't find this syntax at https://stripe.com/docs/api/checkout/sessions/create#create_checkout_session-success_url )

Also could we write the expression without f-strings to make it easier to see what the resulting string would be like?

Reply:
I got rid of this